src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,11 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Lottery ball1 = new Lottery();
```

Comment:
Be careful how you name the variables. The class is called `Lottery`, so the variable should have the appropriate name `lottery`.

**ToDo**: use `lottery` name
Also check out the other naming conventions